sdk/core/core-amqp/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 1.1.6 (Unreleased)
+## 1.1.6 (2020-09-02)
 
+- Remove `stream-browserify` dependency. [10425](https://github.com/Azure/azure-sdk-for-js/issues/10425)
```

Comment:
If this is the only change in core-amqp, then I would recommend skipping the release
cc @HarshaNalluru, @richardpark-msft

Reply:
Yes, this is the only change.
I thought event hubs was waiting for this 'fix'.. but that's not the case.
Fine skipping core-amqp release.

Reply:
We'll probably need a change to support SAS keys.

Reply:
Sounds good. Reverted

Reply:
@richardpark-msft what change? Is it in PR?

Reply:
@jeremymeng That is still in progress and we will release core-amqp independently of the other core packages. So, please exclude it from this PR

cc @sadasant

Reply:
Back to just core-http and core-paging